cluster/ci/config/submit-queue/submit_queue.yaml

```diff
@@ -1,133 +1,23 @@
 apiVersion: v1
-kind: Template
```

Comment:
Do you also need to update make targets because of this change?

Reply:
yarp